spec/acceptance/iis_site_spec.rb

```diff
@@ -112,6 +116,7 @@
           ].each_slice(2) do |key, value|
             puppet_resource_should_show(key, value, resource_data)
           end
+          expect(resource_data.stdout).not_to match(%r{'protocol' => 'http'.*'protocol' => 'https'}m)
```

Comment:
The resource sorting only appears to be invoked from insync.  However that's not called during a `puppet resource`.  That comes from the instances method https://github.com/puppetlabs/puppetlabs-iis/blob/master/lib/puppet/provider/iis_site/webadministration.rb#L145-L155

Reply:
Right.  First I add the test, which fails. The next commit is supposed to fix the test, so it passes.

Reply:
Actually, given that it only needs to be sorted when APPLYing the bindings, not reading them, is this test valid?

Reply:
Maybe not.  Probably I should remove the test altogether.  It should fail creation anyway.

Reply:
done.